packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -227,6 +231,19 @@ export type TSelectableSearchInputProps = {
    * [Props from React select was used](https://react-select.com/props)
    */
   selectCustomComponents?: ReactSelectProps['components'];
+
+  /**
+   * Custom action icon to be displayed on the right side of the input.
+   * [Available Icons](/?path=/story/components-icons--all-icons)
```

Comment:
Including this URL is an interesting concept. However, please note that this JS Doc comment will only work in the deployed Storybook context. So unless we use a FQDN this won't work for Github readme readers, TypeScript users, and in [the npm package readme](https://www.npmjs.com/package/@commercetools-uikit/selectable-search-input)

Reply:
Oh no, didn't think about that case :(

I'd rather remove the links then. Thanks!

Reply:
Done.